content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -910,12 +911,23 @@ kubectl delete pod dapi-test-pod --now
 
 # You might already have removed the next set
 kubectl delete configmaps/special-config configmaps/env-config
-kubectl delete configmap -l 'game-config in (config-4,config-5)’
+kubectl delete configmap -l 'game-config in (config-4,config-5)'
+```
+
+Remove the `kustomization.yaml` that you used to create the ConfigMap from generator:
+
+```bash
+rm kustomization.yaml
 ```
 
 If you created a directory `configure-pod-container` and no longer need it, you should remove that too,
 or move it into the trash can / deleted files location.
 
+```bash
+# Confirm that you are no longer need files in the directory, then remove it:
```

Comment:
nit: Do you need this comment? The previous sentence provides the same information.